client/src/components/categorical/index.tsx

```diff
@@ -133,6 +133,41 @@ class Categories extends React.Component<{}, State> {
     }
   };
 
+  /**
+   * Determine if category name is an ontology key.
+   * @param catName - Name of category.
+   * @returns True if category name includes ontology key.
+   */
+  isCategoryNameOntologyKey = (catName: string): boolean =>
+    catName.includes(globals.ONTOLOGY_KEY);
+
+  /**
+   * Categories are included for display if category has more than one category value or categories are not defined
+   * (for the case where category is a string or boolean type).
+   * @param schema - Matrix schema.
+   * @param catName - Name of category.
+   * @returns True if category has more than one category value or categories are not defined.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types -- update on typing of component
```

Comment:
Maybe we should adopt a standard for these types of comments to make it easier to undo the disables once the appropriate component(s) are typed.  cc: @tihuan

Reply:
It should be something easily cmd + f -able.  Either an issue # or the file path that needs to be typed

Reply:
Oh great question 💡 I think special comment type would be useful if we do `schema: any` because `schema` is not typed yet. In this case, I think `schema` is already typed elsewhere, so it's just a matter of using when we type everything in this file?

Also I'm wondering maybe `@typescript-eslint/explicit-module-boundary-types` is not needed, since we have `(...): boolean => `?

Reply:
I have update the `schema` argument type in both [isCategoryDisplayable](https://github.com/chanzuckerberg/single-cell-explorer/pull/193/files#diff-7f6832c58dab222ca673a6841ad0efb0f300c20eee501bee503fe3d40f0c1b44R152) and [isCategoryWritable](https://github.com/chanzuckerberg/single-cell-explorer/pull/193/files#diff-7f6832c58dab222ca673a6841ad0efb0f300c20eee501bee503fe3d40f0c1b44R176) to `Schema`.

Timmy, good q about `@typescript-eslint/explicit-module-boundary-types`. According to typescript-eslint's [supported rules](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules), it looks both the return type *and* argument types are required (in our case, on public methods):

```
Require explicit return and argument types on exported functions' and classes' public class methods
```

Reply:
Ahh TIL! Thanks so much for sharing, @MillenniumFalconMechanic 🙏 SGTM!